int32     platform_id
Point2d[] path_points
Point2d goal
Point2d final_orient_point
